test/functionalTest/cases/0999_update_single_attr_value_by_eid/update_single_attr_value_by_eid_fake_field_cases.test

```diff
@@ -21,7 +21,7 @@
 # VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
```

Comment:
This test has been "recovered" (from .test.DISABLED to .test).

NTC (just a remark)